rotkehlchen/balances/historical.py

```diff
@@ -323,8 +327,14 @@ def _get_events_and_currency(
                 trade_bindings,
             )
             cursor2.execute(
-                f'SELECT {HISTORY_BASE_ENTRY_FIELDS} FROM history_events WHERE {" AND ".join(event_where_clauses)} ORDER BY timestamp',  # noqa: E501
-                event_bindings,
+                f'SELECT {HISTORY_BASE_ENTRY_FIELDS} FROM history_events WHERE {" AND ".join(event_where_clauses)} '  # noqa: E501
+                f'AND NOT ((type=? AND subtype=?) OR (type=? AND subtype=?)) ORDER BY timestamp',
+                event_bindings + [
+                    HistoryEventType.DEPOSIT.serialize(),
```

Comment:
Don't you already have this in another filter? I remember reviewing a PR with that diff. These filters shouldn't they be in sync?